sdcm/utils/libssh2_client/result.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
```

Comment:
Honestly, it's a copy/paste from https://github.com/pyinvoke/invoke/blob/008e93070540be5153a310d2794fb04b23f0194b/invoke/runners.py#L1334

Looks like invoke's license is compatible with AGPL3, so, I think it's enough to put credits to the original work here.


So, it's be good to have

Reply:
It is. Since we are havily relay on what runner returns it is necessary to have it implemented same way as it is done in invoke.